js/unison.js

```diff
@@ -52,26 +86,26 @@ Unison = (function() {
   };
 
   var breakpoints = {
-    all : function() {
+    all: function () {
```

Comment:
Looks like this also adds a bunch of stylistic changes. No issue, just makes reviewing rather difficult. Best practice is to issue a PR for each single feature/bug.